lib/interface.js

```diff
@@ -25,9 +25,14 @@ module.exports = function (Target) {
       result = result.catch((err) => {
         err.originalStack = err.stack;
         const firstLine = err.stack.split('\n')[0];
-        this._asyncStack.unshift(firstLine);
+
+        const { error, lines } = this._asyncStack;
```

Comment:
Shouldn't the comment of
```
 // a hack to get a callstack into the client code despite this
 // a hack to get a callstack into the client code despite this
 // node.js bug https://github.com/nodejs/node/issues/11865
```
be moved here then?

Reply:
Probably duplicated? Now it places at initialisation

Reply:
So which part relates to working around that particular Node issue? Reference should be there, or both, if it applies to both :)

Reply:
done